keel-titus-api/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/TitusServerGroup.kt

```diff
@@ -1,35 +1,12 @@
-/*
```

Comment:
nit: You have new/moved files above with the copyright. I'm not sure what the current expectation is, TBH, but we should be consistent.